### DISCUSSION TITLE
HDMI Audio Output

### DISCUSSION THREAD

**Discussion by getitdonestudio** [mikebrady/shairport-sync#1595] [Q&A]
I've been trying to change the audio output to one of the two HDMI-Ports of my Raspi 4B by changing the configuration to the following (as written in AdjustingSync.md)

`alsa =
{
output_device = "hw:vc4hdmi0"; 
mixer_control_name = "PCM"; 
...
};`

--with-alsa configuration flag was enabled.

The only audio output I get is still through headphone jack - does anyone have an idea how to resolve this?

Thanks!

**Comment by mikebrady** [comment]
Thanks,. There is a setting in `raspi-config`  to get the Pi to output to HDMI.
